tests/unit/server/migrations/migrate_v12.test.js

```diff
@@ -0,0 +1,246 @@
+        sndFakeMinute = {
+            _id: SND_MIN_ID,
+            topics: [{
+                _id: '#T01',
+                infoItems:[{
+                    _id: '#I01',
+                    details: [{text: 'd1'}, {text:'d22'}, {text:'d6'}]
+                },{
+                    _id: '#I02',
+                    details: [{text: 'd3'}, {text:'d4'}]
+                }]
+            }, {
+                _id: '#T02',
+                infoItems:[{
+                    _id: '#I03',
+                    details: [{text: 'd5'}]
+                }]
+            }],
+            nextMinutes: () => {
+                return false;
+            },
+            previousMinutes : () => {
+                return firstFakeMinute;
+            }
+        };
+
+        firstFakeMinute = {
+            _id: FIRST_MIN_ID,
+            topics: [{
+                _id: '#T01',
+                infoItems:[{
+                    _id: '#I01',
+                    details: [{text: 'd1'}, {text:'d22'}]
+                },{
+                    _id: '#I02',
+                    details: [{text: 'd3'}, {text:'d4'}]
+                }]
+            }],
+            nextMinutes: () => {
+                return sndFakeMinute;
+            },
+            previousMinutes: () => {
+                return false;
+            }
+        };
+
+        fakeMeetingSeries = {
+            _id: '#MS01',
+            topics: [{
+                _id: '#T01',
+                infoItems:[{
+                    _id: '#I01',
+                    details: [{text: 'd1'}, {text:'d22'}, {text:'d6'}]
+                },{
+                    _id: '#I02',
+                    details: [{text: 'd3'}, {text:'d4'}]
+                }]
+            }, {
+                _id: '#T02',
+                infoItems:[{
+                    _id: '#I03',
+                    details: [{text: 'd5'}]
+                }]
+            }],
+            openTopics: [{
+                _id: '#T02',
+                infoItems:[{
+                    _id: '#I03',
+                    details: [{text: 'd5'}]
+                }]
+            }, {
+                _id: '#T01',
+                infoItems:[{
+                    _id: '#I01',
+                    details: [{text: 'd1'}, {text:'d22'}, {text:'d6'}]
+                },{
+                    _id: '#I02',
+                    details: [{text: 'd3'}, {text:'d4'}]
+                }]
+            }],
+            firstMinutes: () => {
+                return firstFakeMinute;
+            }
+        };
+
+        MeetingSeriesCollection.find = () => {
+            return [fakeMeetingSeries];
+        };
+
+        MinutesCollection.find = () => {
+            return [firstFakeMinute, sndFakeMinute];
+        };
+    });
+
+    afterEach(function () {
+        MinutesCollection.update.reset();
+        MeetingSeriesCollection.update.reset();
+    });
+
+    describe('#up', function () {
+
+        let checkDetailHasProperty_createdInMinute = detail => {
```

Comment:
How about this:
```
let checkDetailHasProperties = (properties) => {
    return (detail) => {
        properties.forEach(property => expect(detail).to.have.ownProperty(property));
    }
};
```
Then you can call it like this:
```
infoItem.details.forEach(checkDetailHasProperties(['createdInMinute', '_id']));
```
?

With that we can get rid of some code duplicates ;-)